exchanges/stream/websocket_test.go

```diff
@@ -1111,16 +1105,37 @@ func TestFlushChannels(t *testing.T) {
 	// Disable pair and flush system
 	newgen.EnabledPairs = []currency.Pair{currency.NewPair(currency.BTC, currency.AUD)}
 	w.GenerateSubs = func() (subscription.List, error) { return subscription.List{{Channel: "test"}}, nil }
-	err = w.FlushChannels()
-	require.NoError(t, err, "Flush Channels must not error")
+
+	require.ErrorIs(t, w.FlushChannels(), ErrSubscriptionsNotAdded, "FlushChannels should error correctly on no subscriptions added")
+
+	w.Subscriber = func(subs subscription.List) error {
+		for _, sub := range subs {
+			if err := w.subscriptions.Add(sub); err != nil {
+				return err
+			}
+		}
+		return nil
+	}
+
+	require.NoError(t, w.FlushChannels(), "Flush Channels must not error")
```

Comment:
```suggestion
	require.NoError(t, w.FlushChannels(), "FlushChannels must not error")
```

To be consistent with the previous one